src/main_thread/tracks_store/tracks_store.ts

```diff
@@ -481,17 +498,119 @@ export default class TracksStore extends EventEmitter<ITracksStoreEvents> {
       if (
         periodObj.isPeriodAdvertised &&
         trackObj.dispatcher !== null &&
-        !trackObj.dispatcher.hasSetTrack() &&
-        trackObj.storedSettings !== undefined
```

Comment:
Wait why are we removing this? I guess (though I don't remember why) it as placed on purpose at a guard and it has different semantics than `nul`